include/script_api/macros.h

```diff
@@ -595,13 +595,23 @@
         EVT_END \
     }
 
+// only set perspective to standard values
+#define EVT_SETUP_CAMERA_MINIMAL() \
+    EVT_CALL(SetCamPerspective, CAM_DEFAULT, 3, 25, 16, 4096) \
 
 #define EVT_SETUP_CAMERA_DEFAULT() \
     EVT_CALL(SetCamPerspective, CAM_DEFAULT, 3, 25, 16, 4096) \
     EVT_CALL(SetCamBGColor, CAM_DEFAULT, 0, 0, 0) \
     EVT_CALL(SetCamEnabled, CAM_DEFAULT, TRUE) \
 
 #define EVT_SETUP_CAMERA_NO_LEAD() \
+    EVT_CALL(SetCamPerspective, CAM_DEFAULT, 3, 25, 16, 4096) \
+    EVT_CALL(SetCamBGColor, CAM_DEFAULT, 0, 0, 0) \
+    EVT_CALL(SetCamEnabled, CAM_DEFAULT, TRUE) \
+    EVT_CALL(SetCamLeadPlayer, CAM_DEFAULT, FALSE) \
+
+// similar to previous, but ordered differently
+#define EVT_SETUP_CAMERA_ODD_NO_LEAD() \
     EVT_CALL(SetCamPerspective, CAM_DEFAULT, 3, 25, 16, 4096) \
     EVT_CALL(SetCamBGColor, CAM_DEFAULT, 0, 0, 0) \
     EVT_CALL(SetCamLeadPlayer, CAM_DEFAULT, FALSE) \
```

Comment:
We shouldn't have a trailing `\` on the final lines of these macros I don't think